src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private User user = new User();
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @Before
+    public void setUser() {
+        user.setAge(19);
+        user.setLogin("Genareb");
+        user.setPassword("asdfgh");
+    }
+
+    @Test
+    void register_nullUser_notOk() throws RuntimeException {
+        try {
+            registrationService.register(null);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown in user == null case");
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        setUser();
+        user.setAge(null);
+        try {
+            registrationService.register(user);
+        } catch (RuntimeException e) {
+            return;
+        }

```

Comment:
<img width="888" alt="Screen Shot 2022-10-13 at 6 43 42 PM" src="https://user-images.githubusercontent.com/90692308/195643261-20ee17cc-0517-43ee-9961-5b35c0b6b5fb.png">
Here is an example of how to test cases where an exception will be thrown. Use this option instead of the try/catch block